src/Bundle/Controller/ResourceController.php

```diff
@@ -251,7 +251,7 @@ public function updateAction(Request $request): Response
         $configuration = $this->requestConfigurationFactory->create($this->metadata, $request);
 
         $this->isGrantedOr403($configuration, ResourceActions::UPDATE);
-        $resource = $this->findOr404($configuration);
+        $resource = $this->findOrCreate($configuration);
```

Comment:
It should probably be conditional, since it's only for PUT requests, the PATCH should return 404 if the resource does not exists.
```
        if ($request->isMethod(Request::METHOD_PUT)) {
            $resource = $this->findOrCreate($configuration);            
        } else {
            $resource = $this->findOr404($configuration);
        }
```